### DISCUSSION TITLE
KO mode?

### DISCUSSION THREAD

**Discussion by ddwhzh** [fredzzhang/upt#38] [Q&A] [answered]
Excuse me, I found that the KO mode result for HICO-DET is provided in your paper, but it isn't provided in the code, could you provide any idea, Thank you.

**Comment by fredzzhang** [comment, marked as answer]
Hi @ddwhzh,

Please refer to the instructions for [`Training and Testing`](https://github.com/fredzzhang/upt#training-and-testing).

To get the results under the known object setting, you have to cache the results in `.mat` form and evaluate it separately. The command to generate cached results can be found in the [`launch_template.sh`](https://github.com/fredzzhang/upt/blob/main/launch_template.sh). For the Matlab evaluation code, refer to the [HO-RCNN](https://github.com/ywchao/ho-rcnn) repo.

Cheers,
Fred.

**Comment by ddwhzh** [comment]
THX
